src/collective/frontpage/behaviors.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+
+from collective.frontpage import _
+from plone.autoform.interfaces import IFormFieldProvider
+from plone.namedfile.field import NamedBlobImage
+from plone.supermodel import model
+from zope import schema
+from zope.interface import alsoProvides
+from zope.interface import Interface
+
+
+class ISectionFields(model.Schema):
```

Comment:
Should we name the behaviors based on their target content type or based on their usage? Let's split this up into IColorFields and IImageFields. I know we will have another behavior IReferencedImageFields to keep a relation to an image object instead of uploading it.

Reply:
Instead of moving the fields out, I would create a entirely separate behavior when we need it. ISectionFields implements a set of fields (with descriptions) that work with each other, they should not be split out, this makes no sense in my opinion.